docs/get-started.md

```diff
@@ -11,13 +11,38 @@ weight=-70
 
 # Get started with Docker Machine and a local VM
 
-Let's take a look at using `docker-machine` for creating, using, and managing a
-Docker host inside of <a href=
-"https://www.virtualbox.org/" target="_blank">VirtualBox</a>.
+Let's take a look at using `docker-machine` to create, use and manage a
+Docker host inside of a local virtual machine.
+
+## Prerequisite Information
+
+With the advent of [Docker for Mac](/docker-for-mac/index.md) and [Docker for Windows](/docker-for-windows/index.md) as replacements for [Docker Toolbox](/toolbox/overview.md), there are some new considerations for creating local machines with `docker-machine create`.
+
+The new solutions come with their own native virtualization solutions rather than using Oracle VirtualBox. In the case of Docker for Windows, you'll use a different driver to create machines.
+
+Like Toolbox, both Docker for Mac and Docker for Windows automatically include the newest version of Docker Machine, so when you install either of these, you still get `docker-machine` on your system.
+
+#### If you are using Docker for Windows
+
+Docker for Windows uses [Microsoft Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/windows_welcome) for virtualization, and Hyper-V and Oracle VirtualBox are incompatible. You cannot run the two solutions simultaneously. But you can still use `docker-machine` to create local VMs by using the Microsoft Hyper-V driver.
 
-## Prerequisites
+* If you are using Docker for Windows, the only prequisite is to have Docker for Windows installed. You will use the Microsoft `hyperv` driver to create local machines. (Hyper-V driver reference is [here](drivers/hyper-v.md).)
+
+#### If you are using Docker for Mac
+
+Docker for Mac uses [HyperKit](https://github.com/docker/HyperKit/), a lightweight OS X virtualization solution built on top of the [Hypervisor.framework](https://developer.apple.com/reference/hypervisor) in OS X 10.10 Yosemite and higher.
+
+Currently, there is no `docker-machine create` driver for HyperKit, so you will use `virtualbox` driver to create local machines. (VirtualBox driver reference is [here](drivers/virtualbox.md).) Note that you can run both HyperKit and Oracle VirtualBox on the same system. To learn more, see [Docker for Mac vs. Docker Toolbox](/docker-for-mac/docker-toolbox/).
+
+* Make sure you have <a href="https://www.virtualbox.org/wiki/Downloads" target="_blank">the latest VirtualBox</a> correctly installed
+on your system (either as part of an earlier Toolbox install, or manual install).
+
+#### If you are using Docker Toolbox
+
+* If you are using Docker Toolbox exclusivley on either Mac or Windows, you will use the `virtualbox` driver to create a local machine based on Oracle <a href=
```

Comment:
On condition where this may not apply is when someone is using Windows with Hyper-V but an older version of Windows, say Windows 8 pro. That would mean they couldn't use Virtualbox or Docker for Windows, but could still use the hyperv driver for Machine.